src/serial/openvtx.ts

```diff
@@ -15,6 +15,38 @@ const RST_MAGIC = ["R".charCodeAt(0), "S".charCodeAt(0), "T".charCodeAt(0)];
 export class OpenVTX {
   private constructor(private serial: Serial) {}
 
+  public static async listenForBootloader(serial: Serial, useSmartAudio: Boolean) {
+
+    const ovtx = new OpenVTX(serial);
+
+    if (useSmartAudio){
+      await ovtx.serial.connect({
+        baudRate: 4800,
+        stopBits: 2,
+      });
+    }
+    
+    let tries = 0;
+    while (tries <= 3) {
```

Comment:
nit-pick, this could be a for with the error being thrown after the loop.

Reply:
yeah, might be a good idea to inform user.

Reply:
A count down message for every try?